org.prismlauncher.PrismLauncher.yml

```diff
@@ -0,0 +1,90 @@
+app-id: org.prismlauncher.PrismLauncher
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
```

Comment:
5.15-22.08 maybe? This one might go EOL sooner than you might want.

Reply:
done